storm-server/src/main/java/org/apache/storm/scheduler/resource/strategies/scheduling/ConstraintSolverStrategy.java

```diff
@@ -41,6 +41,9 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+// When making changes to scheduling strategy, be sure to run performance performance unit test
+// TestResourceAwareScheduler::testLargeFragmentedClusterScheduling
```

Comment:
I don't think this is a good practice to link to a unit test. And we don't need to worry about reminding developers to run a specific unit test because running unit tests is something that will just happen. Developers will run all the related unit tests or at least let Travis run the unit tests.

Reply:
removed